csrc/tensor_view.cpp

```diff
@@ -240,69 +240,6 @@ std::string TensorView::toInlineString(int indent_size) const {
   return toString(indent_size);
 }
 
-void TensorView::convertRfactorToRootDomain() {
```

Comment:
Is this change completely unrelated with the propagation of concretized vals?

Reply:
Yes. At first I thought it was going to be necessary. I realized it was not necessary for this PR, but it is a little faster and reduces the TensorView interface so I figured I would keep it. I can split it into another PR if you prefer.